ursa/local_manager.py

```diff
@@ -235,7 +235,7 @@ def split_graph(self, graph_id):
 
 
 @ray.remote
-def _get_local_edges(edges_oid, buffer_oid):
-    remotes = ray.get(edges_oid)
-    remotes.append(buffer_oid)
+def _get_local_edges(edges, buf):
```

Comment:
Are we missing the docstring for this function?

Reply:
Yes, I will add